CvGameCoreDLL/CvPlayerAI.cpp

```diff
@@ -5042,8 +5042,7 @@ TechTypes CvPlayerAI::AI_bestTech(int iMaxPathLength, bool bIgnoreCost, bool bAs
 									if (iI == MYTHOLOGY || iI == DIVINATION || iI == CEREMONY)
 										iValue /= 5;
 								case GREECE:
-									if (iI == THEOLOGY)

```

Comment:
Can we keep C++ style curly brackets (isolated line for opening and closing) please?